common/lib/capa/capa/responsetypes.py

```diff
@@ -3012,8 +3005,7 @@ def get_answers(self):
         except Exception as err:  # pylint: disable=broad-except
             log.error('Error %s', err)
             if self.capa_system.DEBUG:
-                msg = '<span class="inline-error">%s</span>' % str(
-                    err).replace('<', '&lt;')
+                msg = HTML('<span class="inline-error">{}</span>').format(Text(err))
```

Comment:
FYI: You don't need the `Text` call here, but it is fine to leave it if you want that to be more explicit.